### PR TITLE
Homework 2.2

### PR DESCRIPTION
**Pull request by MaliukDaria**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,25 @@
...
+        stopWork(excavator);
+        stopWork(bulldozer);
+        stopWork(truck);
+    }
+
+    public static void doWork(Machine machine) {
+        machine.doWork();
+    }
+
+    public static void stopWork(Machine machine) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:33]: this method is redundant in MainApp class, let's remove them

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -12,5 +12,25 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:16]: be attentive with [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)
- Use abstract class to create Track, Bulldozer and Excavator instances.

Example:
Animal cat = new Cat();

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (MaliukDaria) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 2 above)
Comment (MaliukDaria) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**